### PR TITLE
docs(blog): created a release 2.8 blog post

### PR DESCRIPTION
**Pull request by lukidzi**
<Explain your change!>

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits)

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: app/_posts/2024-06-25-kuma-2-8-0.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+date: 2024-06-25
+headerImage: /assets/images/blog/test-header.jpg
+tags:
+  - Release
+---
+
+We’re excited to announce the release of Kuma 2.8, featuring new and exciting capabilities such as `MeshExternalService`, `MeshPassthrough` policy, along with numerous improvements and groundwork for future developments.
+
+# Notable features
+* A new `MeshExternalService` resource that overcomes existing issues with the `ExternalService` resource.
```
Comment (lahabana) [inline app/_posts/2024-06-25-kuma-2-8-0.md:13]: Limitations not issues

File: app/_posts/2024-06-25-kuma-2-8-0.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+## MeshExternalService
+The `ExternalService` resource struggled with various issues, leading us to develop the more powerful [`MeshExternalService`](/docs/2.8.x/networking/meshexternalservice/) resource. This brand-new resource enables you to add external endpoints to your mesh and overcome the limitations of the `ExternalService`. `MeshExternalService` will eventually replace `ExternalService` and is currently in the alpha phase.
+
+![MeshExternalService tab view](/assets/images/blog/kuma_2_8_mes.png)
+
+Additionally, you can see status information about the hostname and address of the `MeshExternalService`.
+
+![MeshExternalService status](/assets/images/blog/kuma_2_8_mes_status.png)
+
+## MeshPasssthrough
```
Comment (lahabana) [inline app/_posts/2024-06-25-kuma-2-8-0.md:74]: Typo

File: app/_posts/2024-06-25-kuma-2-8-0.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+tags:
+  - Release
+---
+
+We’re excited to announce the release of Kuma 2.8, featuring new and exciting capabilities such as `MeshExternalService`, `MeshPassthrough` policy, along with numerous improvements and groundwork for future developments.
+
+# Notable features
+* A new `MeshExternalService` resource that overcomes existing limitations with the `ExternalService` resource.
+* A new `MeshPassthrough` policy that allows exposing external endpoints for rich clients through the mesh and support for wildcard domains.
+* HostnameGenerator provides a way to generate custom domains for your `MeshExternalServices`, offering enhanced security.
```
Comment (slonka) [inline app/_posts/2024-06-25-kuma-2-8-0.md:15]: This sounds a bit weird 🤔 (the second part) and what security benefits does it offer?

```suggestion
* HostnameGenerator provides a way to generate custom domains for your `MeshExternalServices`.
```

File: app/_posts/2024-06-25-kuma-2-8-0.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  match:
+    type: HostnameGenerator
+    port: 80
+    protocol: http
+  endpoints:
+  - address: httpbin.org
+    port: 80
+```
+
+In a brand new tab in the GUI, we can now see the generated domain address for our new `MeshExternalService`.
```
Comment (slonka) [inline app/_posts/2024-06-25-kuma-2-8-0.md:61]: "brand new" repeated

```suggestion
In a new tab in the GUI, you can now see the generated domain address for `MeshExternalServices`.
```

File: app/_posts/2024-06-25-kuma-2-8-0.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+![MeshExternalService view with a custom domain provided by the HostnameGenerator](/assets/images/blog/kuma_2_8_mes_hostname.png)
+
+## MeshExternalService
+The `ExternalService` resource struggled with various issues, leading us to develop the more powerful [`MeshExternalService`](/docs/2.8.x/networking/meshexternalservice/) resource. This brand-new resource enables you to add external endpoints to your mesh and overcome the limitations of the `ExternalService`. `MeshExternalService` will eventually replace `ExternalService` and is currently in the alpha phase.
```
Comment (slonka) [inline app/_posts/2024-06-25-kuma-2-8-0.md:66]: it uses "brand-new" once more and I don't think "to struggle" is the right verb when it comes to policies 🤔 

```suggestion
The `ExternalService` resource had some shortcomings, leading us to develop a more powerful [`MeshExternalService`](/docs/2.8.x/networking/meshexternalservice/) resource. This new resource enables you to add external endpoints to your mesh and overcome the limitations (should we name them? I feel like here we're just repeating what was said in the previous paragraph) of the `ExternalService`. `MeshExternalService` is currently in the alpha phase and will eventually replace `ExternalService`.
```

File: app/_posts/2024-06-25-kuma-2-8-0.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+![MeshExternalService tab view](/assets/images/blog/kuma_2_8_mes.png)
+
+Additionally, you can see status information about the hostname and address of the `MeshExternalService`.
```
Comment (slonka) [inline app/_posts/2024-06-25-kuma-2-8-0.md:70]: we've already mentioned that a couple of lines above:

> In a brand new tab in the GUI, we can now see the generated domain address for our new `MeshExternalService`.

File: app/_posts/2024-06-25-kuma-2-8-0.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+![MeshExternalService status](/assets/images/blog/kuma_2_8_mes_status.png)
+
+## MeshPassthrough
+
+Some of the issues with `ExternalService` didn't align with the new design of `MeshExternalService`. As a result, we've introduced a new policy called [`MeshPassthrough`](https://kuma.io/docs/2.8.x/policies/meshpassthrough). This policy facilitates communication with external endpoints for rich clients and offers the flexibility to enable or disable [passthrough mode](/docs/2.8.x/networking/non-mesh-traffic/#outgoing) for individual sidecars. Learn more about passthrough mode.
```
Comment (slonka) [inline app/_posts/2024-06-25-kuma-2-8-0.md:76]: should passthrough mode be a link?

should we add more info about what "rich clients" are?

```suggestion
Some of the issues with `ExternalService` didn't align with the new design of `MeshExternalService`. As a result, we've introduced a new policy called [`MeshPassthrough`](https://kuma.io/docs/2.8.x/policies/meshpassthrough). This policy facilitates communication with external endpoints for rich clients and offers the flexibility to enable or disable [passthrough mode](/docs/2.8.x/networking/non-mesh-traffic/#outgoing) for individual sidecars. Learn more about [passthrough mode](?).
```

**Review by slonka — COMMENTED** [review]
Just a first pass, I think I'll take another look at it tomorrow.

File: app/_posts/2024-06-25-kuma-2-8-0.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
+![MeshExternalService tab view](/assets/images/blog/kuma_2_8_mes.png)
+
+Additionally, you can see status information about the hostname and address of the `MeshExternalService`.
+
+![MeshExternalService status](/assets/images/blog/kuma_2_8_mes_status.png)
+
+## MeshPassthrough
+
+Some of the issues with `ExternalService` didn't align with the new design of `MeshExternalService`. As a result, we've introduced a new policy called [`MeshPassthrough`](https://kuma.io/docs/2.8.x/policies/meshpassthrough). This policy facilitates communication with external endpoints for rich clients and offers the flexibility to enable or disable [passthrough mode](/docs/2.8.x/networking/non-mesh-traffic/#outgoing) for individual sidecars. Learn more about [passthrough mode](?).
```
Comment (lahabana) [inline app/_posts/2024-06-25-kuma-2-8-0.md:76]: The link at the end is incomplete

File: app/_posts/2024-06-25-kuma-2-8-0.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+  match:
+    type: HostnameGenerator
+    port: 80
+    protocol: http
+  endpoints:
+    - address: httpbin.org
+      port: 80
+```
+
+In the coming releases of Kuma we'll be providing a default `HostnameGenerator` and potential extensions for integrating with complex infrastructure like aws lambda.
```
Comment (slonka) [inline app/_posts/2024-06-25-kuma-2-8-0.md:63]: nitpick:

```suggestion
In the coming releases of Kuma we'll be providing a default `HostnameGenerator` and potential extensions for integrating with complex infrastructure like AWS lambda.
```

**Review by slonka — APPROVED** [review]

**Pull request merged**